assets/queries/crossplane/aws/cloudfront_without_waf/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "6d19ce0f-b3d8-4128-ac3d-1064e0f00494",
+  "queryName": "Cloudfront Without WAF",
```

Comment:
I suggest standardize "Cloudfront". In the previous query, it is as "CloudFront". Maybe we can verify all the queries and decide to standardize as "CloudFront"